problem-solver/py/modules/messageProcessingModule/WeatherAgent.py

```diff
@@ -0,0 +1,76 @@
+"""
+This code creates some test agent and registers until the user stops the process.
+For this we wait for SIGINT.
+"""
+
+import logging
+import os
+from sc_client.models import ScAddr, ScLinkContentType
+
+from sc_client.constants import sc_types
+
+from sc_kpm import ScAgentClassic, ScModule, ScResult, ScServer
+from sc_kpm.utils import create_link, get_link_content_data
+from sc_kpm.utils.action_utils import (
+    create_action_answer,
+    finish_action_with_status,
+    get_action_arguments,
+)
+from sc_kpm.utils import check_edge, create_edge
+from sc_kpm.utils import create_link
+
+from sc_kpm.utils import get_element_by_role_relation, get_element_by_norole_relation
+
+import requests
+
+from sc_kpm import ScKeynodes
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s | %(levelname)s | %(name)s | %(message)s", datefmt="[%d-%b-%y %H:%M:%S]"
+)
+
+logging.basicConfig(level=logging.INFO)
+
+
+class WeatherAgent(ScAgentClassic):
+    def on_event(self, event_element: ScAddr, event_edge: ScAddr, action_element: ScAddr) -> ScResult:
+        result = self.run(action_element)
+        is_successful = result == ScResult.OK
+        finish_action_with_status(action_element, is_successful)
+        self.logger.info("Agent finished %s", "successfully" if is_successful else "unsuccessfully")
+        return result
+    
+    def run(self, action_node: ScAddr) -> ScResult:
+        self.logger.info("Agent began to run")        
+
+        messageAddr = get_action_arguments(action_node, 1)[0]
+
+        show_weather_answer_phrase = ScKeynodes.resolve("show_weather_answer_phrase", None)
```

Comment:
Check python codestyle in documentation. Use lowercase and do not pass as much empty lines